modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -728,7 +745,7 @@ object SeqexecEngine extends SeqexecConfiguration {
 
     // TODO: Review initialization of EPICS systems
     @SuppressWarnings(Array("org.wartremover.warts.Throw"))
-    def initEpicsSystem[T](sys: EpicsSystem[T], tops: Map[String, String]): IO[Unit] =
+    def initEpicsSystem(sys: EpicsSystem[_], tops: Map[String, String]): IO[Unit] =
```

Comment:
Are these equivalent?

Reply:
Yes. If the type parameter isn't mentioned again it can be existential. I prefer it this way because it's instantly clear that the type argument is irrelevant.